poetry/utils/exporter.py

```diff
@@ -96,6 +99,9 @@ def _export_requirements_txt(
                 line = "{}".format(package.source_url)
                 if package.develop and package.source_type == "directory":
                     line = "-e " + line
+            elif no_freeze:
+                dependency = package.to_dependency()
+                line = package.name
```

Comment:
afaics, this strips the dependency to just being the naked package name, sans the pyproject.toml specification.

Reply:
+1, it would be great to move soft deps like `1.0.*` as is too.